src/main/resources/openapi/events.yaml

```diff
@@ -83,6 +83,48 @@ paths:
                 $ref: './problem.yaml'
 
   /courtlogs:
+    get:
+      tags:
+        - Event
+      summary: An Endpoint which allows users to request courtlogs
```

Comment:
wonder should we put `(events)` in here also, to make it obvious what we mean by courtlogs